.travis.yml

```diff
@@ -139,9 +139,10 @@ jobs:
       - (cd / && go get github.com/mattn/goveralls)
       script:
       - make test-unit
-      - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
       - make test-e2e-go
       - make test-integration
+      after_success:
+      - $GOPATH/bin/goveralls -service=travis-ci -coverprofile=coverage.out -repotoken=$COVERALLS_TOKEN
```

Comment:
Does this command actually run the unit tests again? Or does it just send the results that `make test-unit` already generated?

See: https://github.com/mattn/goveralls#continuous-integration

Is there a separate way to just send the results without re-running tests?

Reply:
@joelanford, 

The command will NOT run the tests again. See: https://travis-ci.org/github/operator-framework/operator-sdk/jobs/694214934#L1532

This line only use the github.com/mattn/goveralls to sent the report (coverage.out) to the coveralls.io.